plugins/catalog-backend-module-backstage-openapi/README.md

```diff
@@ -2,28 +2,30 @@
 
 ## Summary
 
-This module installs an entity provider that exports a single entity, your Backstage instance documentation, which merges as many backend plugins as you have defined in the config value `catalog.providers.openapi.plugins`.
+This module installs an entity provider that exports a single entity, your Backstage instance documentation, which merges as many backend plugins as you have defined in the config value `catalog.providers.backstageOpenapi.plugins`.
 
 ## Notes
 
 - **This only works with the new backend system.**
 
 ## Installation
 
-To your new backend file, add
+To your new backend file, add:
 
 ```ts title="packages/backend/src/index.ts"
 backend.add(
   import('@backstage/plugin-catalog-backend-module-backstage-openapi'),
 );
 ```
 
-Add a list of plugins to your config like,
+Add a list of plugins (and optionally, a custom name/title) to your config like:
 
 ```yaml title="app-config.yaml"
 catalog:
   providers:
-    openapi:
+    backstageOpenapi:
+      name: 'internal_backstage_api' # Optional
+      title: 'Backstage API' # Optional
```

Comment:
How about something like `backstageOpenapi.entityOverrides`, where you can pass any entity override data you'd like to? It would then be merged back into the base entity that we create down below. You could use it like so,
```
backstageOpenapi:
    entityOverrides:
        metadata:
            name: 'my name'
            title: 'my title'
        spec:
            owner: 'my team'
```

and those properties would be overwritten as well.

Thinking here is I was also setting defaults for some of the spec fields like owner and lifecycle that may not be true for your instance, or you may want to add additional fields and this would let you more easily do that.

Reply:
Thank you for the super quick feedback! This could actually be quite useful on our team as well to expand the configuration options - will implement and update the PR 👍